### DISCUSSION TITLE
Poetry dependency not being updated

### DISCUSSION THREAD

**Discussion by shadycuz** [renovatebot/renovate#18658] [Request Help]
I have a `pyproject.toml` file with this dependency `pytest = "^7.1.2"`.  About 4 or 5 days ago `pytest` release `7.2` but I have not gotten a PR with the update yet. 

My repo is here https://github.com/DontShaveTheYak/cf2tf

I'm not sure if I'm just being impatient or if I have something misconfigured.

**Comment by rarkins** [comment]
Look up rangeStrategy docs - you probably want to change it

**Reply by shadycuz** [reply]
Is it supported for `poetry`? https://github.com/renovatebot/renovate/issues/7358
